lib/Conversion/TorchToArith/TorchToArith.cpp

```diff
@@ -192,6 +192,7 @@ class ConvertTorchTensorLiteralOp
           auto shapedType =
               RankedTensorType::get(type.getShape(), builtinTensorElemTy);
           AsmResourceBlob *blob = elements.getRawHandle().getBlob();
+          (void)blob;
```

Comment:
Let's remove `blob` instead since it is no longer used

Reply:
ok sure